packages/client-adapters/src/adapters/base-adapter.js

```diff
@@ -27,6 +28,11 @@ class BaseAdapter {
     this.errors.push(error)
   }
 
+  addCedictError (errorCode, message) {

```

Comment:
see my comment below .. I think this should be something generic and not specific to Cedict

Reply:
Agree